### PR TITLE
Fix Group Variable Changes in Hyprland Module

### PR DESCRIPTION
**Pull request by Airradda**
This resolve the issues introduce in Hyprland PR [#3522](https://github.com/hyprwm/Hyprland/pull/3522) which moved group variables out of general and into their own [section](https://wiki.hyprland.org/Configuring/Variables/#group).

### CODE REVIEW

**Review by danth — APPROVED** [review]
Ready to merge once 0.31 comes through on the unstable branch.

**Comment by Airradda** [comment]
Looks like NixOS/nixpkgs#261451 got merged to master. I believe we are just waiting on the merge/rebase of [master to nixpkgs-unstable](https://nixpk.gs/pr-tracker.html?pr=261451), correct?

**Comment by danth** [comment]
Yes, that should take a few days at most.

**Pull request merged**
